### PR TITLE
Use environment var for searching for programs

### PR DESCRIPTION
**Pull request by iwootten**
Lookup network tool path for programs from an environment variable over an assumed relative path to the current file.

### CODE REVIEW

**Comment by tokejepsen** [comment]
That's a good idea :)

File: pipeline_plugins/resource/hook/celaction.py [hunk 1]
```diff
@@ -9,8 +9,7 @@
 import _winreg
 
 if __name__ == "__main__":
```
Comment (tokejepsen) [inline pipeline_plugins/resource/hook/celaction.py]: We don't actually need this when run through ftrack-connect.
Are you using this?

Comment (iwootten) [reply pipeline_plugins/resource/hook/celaction.py]: Nope, just noticed the path reference and felt it should all be cleaned up at the same time.

Comment (tokejepsen) [reply pipeline_plugins/resource/hook/celaction.py]: Should we get rid of the ```if __name__ == "__main__":``` part of the code?

File: pipeline_plugins/resource/hook/celaction_network.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 import re
 
 if __name__ == "__main__":
```
Comment (tokejepsen) [inline pipeline_plugins/resource/hook/celaction_network.py]: We can remove this as well.

File: pipeline_plugins/resource/hook/generate_titles.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 logger = logging.getLogger()
 
 if __name__ == '__main__':
```
Comment (tokejepsen) [inline pipeline_plugins/resource/hook/generate_titles.py]: We can remove this as well.

**Comment by tokejepsen** [comment]
Looks good :)

**Pull request merged**